### ISSUE TITLE
Incompatible with Effortless Building

### ISSUE THREAD

**Issue by Atteri02** [qouteall/ImmersivePortalsMod#1226]
### According to your testing, does the issue occur with ONLY Immersive Portals mod and Farbic API?

Yes

### Describe the issue

[Describe the issue.](https://www.curseforge.com/minecraft/mc-mods/effortless-building)
![Untitled](https://user-images.githubusercontent.com/104736879/209700848-e9f419af-33ed-40b0-b6d8-3b0f791e4ec0.png)

 Will crash the game with the error code in the picture if you try to destroy blocks with the mod (in survival)
 Basically if it needs to drop any blocks because it works fine in creative.
 Playing in singleplayer
 
 Wouldnt let me put the log as text
[latest.log](https://github.com/iPortalTeam/ImmersivePortalsMod/files/10309846/latest.log)

**Comment by qouteall** [comment]
This is probably an issue of effortless building. It seems that effortless building does entity-related operations in non-server thread. I need the server log to know the details. You can report this in effortless building.
